src/LookupModel.php

```diff
@@ -57,7 +57,7 @@ public function __construct(array $lookup)
         $this->latitude = $this->getOrNull('latitude', $lookup);
         $this->longitude = $this->getOrNull('longitude', $lookup);
         $timezone = $this->getOrNull('timezone', $lookup);
-        if (!is_null($timezone)) {
+        if (!is_null($timezone) && in_array($timezone, timezone_identifiers_list())) {
```

Comment:
The only reason I say go for @Darunada's solution is because there may come a time when you don't tell us that something might have changed. If you hardcode that value you take on the responsibility to tell everyone, all users of the API, all SDK developers that there is a new value and they have to update their implementations. Using in_array prevents you from having to do that in this SDK instance.